images/argo/metadata.hcl

```diff
@@ -0,0 +1,9 @@
+name                = "argo"
+image               = "cgr.dev/chainguard/argo"
+logo                = "https://storage.googleapis.com/chainguard-academy/logos/argo.svg"
+endoflife           = ""
+console_summary     = ""
+short_description   = "Images for working with [Argo workflows](https://argoproj.github.io/argo-workflows/)"
+compatibility_notes = ""
+readme_file         = "README.md"
```

Comment:
Will this ever not be README.md?

Reply:
I wasn't sure about this - thinking about the future where it might be a set of files depending on the different images that things like cert-manager might create.

But maybe it would be better to just make it a constant for now and we can revisit a future monopod if this needs to change?